### PR TITLE
Wording updates

### PR DESCRIPTION
**Pull request by Bettelstab**

### CODE REVIEW

File: src/lib/Constants.ts [hunk 1]
```diff
@@ -2,9 +2,9 @@
  * Sorted by context and alphabetically
  */
 
-// Accounts
-export const ACCOUNT_DEFAULT_LABEL_KEYGUARD = 'Standard Account';
-export const ACCOUNT_DEFAULT_LABEL_LEDGER = 'Ledger Account';
+// Addresses
+export const ACCOUNT_DEFAULT_LABEL_KEYGUARD = 'Standard Address';
+export const ACCOUNT_DEFAULT_LABEL_LEDGER = 'Ledger Address';
```
Comment (sisou) [inline src/lib/Constants.ts]: Rename the constant variable name as well?

File: src/lib/Constants.ts [hunk 2]
```diff
@@ -13,11 +13,11 @@ export const TX_VALIDITY_WINDOW = 120;
...
-// Wallet
+// Accounts
 export const WALLET_BIP32_BASE_PATH_KEYGUARD = `m/44'/242'/0'/`;
-export const WALLET_DEFAULT_LABEL_KEYGUARD = 'Keyguard Wallet';
-export const WALLET_DEFAULT_LABEL_LEDGER = 'Ledger Wallet';
-export const WALLET_DEFAULT_LABEL_LEGACY = 'Legacy Wallet';
+export const WALLET_DEFAULT_LABEL_KEYGUARD = 'Keyguard Account';
+export const WALLET_DEFAULT_LABEL_LEDGER = 'Ledger Account';
+export const WALLET_DEFAULT_LABEL_LEGACY = 'Legacy Account';
 export const WALLET_MAX_ALLOWED_ACCOUNT_GAP = 20;
```
Comment (sisou) [inline src/lib/Constants.ts]: Rename the constant variable names as well? Remember to rename the old `ACCOUNT_*` variables first, or you will not know which were the old and which are the new `ACCOUNT_*` variables.

File: src/lib/CookieDecoder.ts [hunk 3]
```diff
@@ -129,7 +129,7 @@ export class CookieDecoder {
         const labelBytes = this.readBytes(bytes, labelLength);
 
         let accountLabel: string;
-        if (labelBytes.length === 0) accountLabel = 'Standard Account';
+        if (labelBytes.length === 0) accountLabel = 'Standard Address';
```
Comment (sisou) [inline src/lib/CookieDecoder.ts]: Use the constant here?

File: src/views/AddAccountSuccess.vue [hunk 4]
```diff
@@ -1,11 +1,11 @@
 <template>
```
Comment (sisou) [inline src/views/AddAccountSuccess.vue]: This file changes also in #132, thus these changes are overwritten.

File: src/views/AddAccountSuccess.vue [hunk 5]
```diff
@@ -37,7 +37,7 @@ export default class AddAccountSuccess extends Vue {
     @State private keyguardResult!: DeriveAddressResult;
 
     private walletLabel: string = '';
-    private accountLabel: string = 'Standard Account';
+    private accountLabel: string = 'Standard Address';
```
Comment (sisou) [inline src/views/AddAccountSuccess.vue]: Use constant.

**Comment by Bettelstab** [comment]
Ready for another round of reviews.

File: src/lib/Constants.ts [hunk 6]
```diff
@@ -14,16 +14,16 @@ export const TX_VALIDITY_WINDOW = 120;
...
-export const WALLET_DEFAULT_LABEL_KEYGUARD = 'Keyguard Wallet';
-export const WALLET_DEFAULT_LABEL_LEDGER = 'Ledger Wallet';
-export const WALLET_DEFAULT_LABEL_LEGACY = 'Legacy Wallet';
-export const WALLET_MAX_ALLOWED_ACCOUNT_GAP = 20;
+// Accounts
+export const ACCOUNT_BIP32_BASE_PATH_KEYGUARD = `m/44'/242'/0'/`;
+export const ACCOUNT_DEFAULT_LABEL_KEYGUARD = 'Keyguard Account';
+export const ACCOUNT_DEFAULT_LABEL_LEDGER = 'Ledger Account';
+export const ACCOUNT_DEFAULT_LABEL_LEGACY = 'Legacy Account';
+export const ACCOUNT_MAX_ALLOWED_ACCOUNT_GAP = 20;
```
Comment (sisou) [inline src/lib/Constants.ts]:
```suggestion
export const ACCOUNT_MAX_ALLOWED_ADDRESS_GAP = 20;
```

File: src/lib/WalletInfoCollector.ts [hunk 7]
```diff
@@ -140,10 +140,10 @@ export default class WalletInfoCollector {
         const existingWalletInfo = await WalletStore.Instance.get(walletId);
         if (existingWalletInfo) return existingWalletInfo;
         const label = walletType === WalletType.LEGACY
-            ? WALLET_DEFAULT_LABEL_LEGACY
+            ? ACCOUNT_DEFAULT_LABEL_LEGACY
             : walletType === WalletType.BIP39
-                ? WALLET_DEFAULT_LABEL_KEYGUARD
-                : WALLET_DEFAULT_LABEL_LEDGER;
+                ? ADDRESS_DEFAULT_LABEL_KEYGUARD
+                : ADDRESS_DEFAULT_LABEL_LEDGER;
```
Comment (sisou) [inline src/lib/WalletInfoCollector.ts]: Should use the _account_ labels:
```diff
-                ? ADDRESS_DEFAULT_LABEL_KEYGUARD
-                : ADDRESS_DEFAULT_LABEL_LEDGER;
+                ? ACCOUNT_DEFAULT_LABEL_KEYGUARD
+                : ACCOUNT_DEFAULT_LABEL_LEDGER;
```

File: src/lib/CookieDecoder.ts [hunk 8]
```diff
@@ -4,7 +4,14 @@ import { WalletInfoEntry, WalletType } from './WalletInfo';
...
 import { AccountInfoEntry } from './AccountInfo';
 import CookieJar from './CookieJar';
-import { LABEL_MAX_LENGTH } from '@/lib/Constants';
+import {
+    ACCOUNT_DEFAULT_LABEL_KEYGUARD,
+    ACCOUNT_DEFAULT_LABEL_LEDGER,
+    ADDRESS_DEFAULT_LABEL_KEYGUARD,
+    LABEL_MAX_LENGTH,
+    ADDRESS_DEFAULT_LABEL_LEDGER,
+    ACCOUNT_DEFAULT_LABEL_LEGACY,
```
Comment (sisou) [inline src/lib/CookieDecoder.ts]: The order of the constants is a bit mixed. The accounts_* and address_* constants should be grouped together.

File: src/lib/CookieDecoder.ts [hunk 9]
```diff
@@ -134,20 +144,26 @@ export class CookieDecoder {
...
-        else accountLabel = Utf8Tools.utf8ByteArrayToString(new Uint8Array(labelBytes));
+        const accountLabel = labelBytes.length === 0
+            ? type === WalletType.BIP39
+                ? ADDRESS_DEFAULT_LABEL_KEYGUARD
+                : ADDRESS_DEFAULT_LABEL_LEDGER
+            : Utf8Tools.utf8ByteArrayToString(new Uint8Array(labelBytes));
 
         // Account address
-        const addressBytes = this.readBytes(bytes, 20 /* Nimiq.Address.SERIALIZED_SIZE */);
+        const addressBytes = this.readBytes(bytes, Nimiq.Address.SERIALIZED_SIZE);
```
Comment (sisou) [inline src/lib/CookieDecoder.ts]: Revert this. The iframe does not have the Nimiq lib loaded, that's why I used the plain number.

**Pull request merged**